app/views/newfromtemplate.html

```diff
@@ -16,6 +16,7 @@
                   {{ emptyMessage }}
                 </div>
                 <div class="osc-form" ng-show="template">
+                  <alerts alerts="alertsTop" hide-close-button="true"></alerts>
```

Comment:
We talked about moving to [toasts](http://www.patternfly.org/pattern-library/communication/toast-notifications/#/api) at one point, which I think would work as they are fixed position & lay on top of the screen.  Maybe we consider again?